applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/Kconfig

```diff
@@ -217,6 +217,14 @@ config BT_AUDIO_SCAN_DELEGATOR
 	  search for a predefined broadcast source. Instead, it will wait for a
 	  broadcast assistant to connect and control.
 
+config BT_AUDIO_BROADCAST_ZBUS_EVT_STREAM_SENT
+	bool "Enable Zephyr bus event for stream sent"
+	depends on TRANSPORT_BIS
```

Comment:
Suggest depends on is removed and this is set to default=n

Reply:
Removed depends on, kconfig are by default set to `n` if no default is given